internal/services/stock_exchange/algorithms.go

```diff
@@ -0,0 +1,340 @@
+package stock_exchange
+
+import (
+	"Tradeasy/config"
+	model "Tradeasy/internal/model/stock_exchange"
+	"fmt"
+	"math"
+	"time"
+)
+
+type OrderId struct {
+	id int
+}
+
+var orderResponse = make(chan OrderResponse)
+
+func UpdateLTP(ltp int, stock string) {
+	currentStock := model.Stocks{}
+	config.DB.Raw("SELECT * FROM stocks WHERE stock_name = ?", stock).Scan(&currentStock)
```

Comment:
build queries with gorm methods.

Reply:
Done a few places. Every query will take a lot of time as there are a lots of them. Also this is working fine.